ironfish-zkp/src/primitives/value_commitment.rs

```diff
@@ -26,6 +29,29 @@ impl ValueCommitment {
         (self.asset_generator.clear_cofactor() * jubjub::Fr::from(self.value))
             + (*VALUE_COMMITMENT_RANDOMNESS_GENERATOR * self.randomness)
     }
+
+    pub fn to_bytes(&self) -> Vec<u8> {
```

Comment:
Why not having a `write()` like other structs?

Also, I might be wrong, but I think the bytes representation of this struct is fixed-size, so this could return an array instead of allocating a vec